docs/manage/scheduled-views/view-list-scheduled-views.md

```diff
@@ -19,13 +19,14 @@ You must have a role that grants you the View Scheduled Views [role capability
     * **Data Forwarding**. If the scheduled view is configured to forward data to an S3 bucket, this column contains the name of the [data forwarding](../data-forwarding/amazon-s3-bucket.md) destination.  
```

Comment:
This doesn't read well. It's not clear which specific column is being referred to.

Reply:
updated.